README.md

```diff
@@ -1,2 +1,19 @@
 # hw01
 Repository for first CFSS homework assignment
+
+Name:Archer Pan  
+File:HW1  
+Due Date:Jan 11th    
+## **Archer Pan's Brief Biography**
+###**Education**
+I'm currently a MAPSS student concentrating on *Behavioral Science*.I'm researching false memeory and foreign language effect under supervision of Dr.Boaz Keysar.  
```

Comment:
You do not have to italicize "behavioral science" when you are typing the code, the two asterisks will take care of it.